docs/content/_index.md

```diff
@@ -48,20 +48,20 @@ of the execution when triggered by a `Pull Request` or a `Push`.
 
 ## Getting Started
 
-The easiest way to get started is to use the `tkn pac` CLI and its [bootstrap](/docs/guide/cli/#commands) command.
+The easiest way to get started is to use the `tkn pac` CLI and its [bootstrap](./docs/guide/cli.md#install) command.
 
-Start downloading and install the tkn-pac CLI following [these instructions](/docs/guide/cli#install) and
+Start downloading and install the tkn-pac CLI following [these instructions](./docs/guide/cli.md#install) and
 while Connected to your cluster run the command :
 
 ```bash
 -$ tkn pac bootstrap
 ```
 
 and follow the questions to get Pipelines as Code installed on your cluster.
-It will then help you to create create a GitHub Application to connect your repositories to Pipelines as Code.
+It will then help you to create a GitHub Application to connect your repositories to Pipelines as Code.
 If you are in a source code project, it will immediately ask you if you want to have a sample `PipelineRun` for `Pipelines as Code`
 
-## Walkthrough video
+## Walk-through video
```

Comment:
spell checker say Walk-through but google says Walkthrough is an `adjective` :D 

![image](https://user-images.githubusercontent.com/98980/162702890-ee926815-c183-4217-b41a-e158d0cef357.png)